src/interfaces/chain.cpp

```diff
@@ -290,10 +285,13 @@ class ChainImpl : public Chain
         auto it = ::mempool.GetIter(txid);
         return it && (*it)->GetCountWithDescendants() > 1;
     }
-    void relayTransaction(const uint256& txid) override
+    bool broadcastTransaction(const CTransactionRef& tx, std::string& err_string, const CAmount& max_tx_fee, bool relay) override
     {
-        CInv inv(MSG_TX, txid);
-        g_connman->ForEachNode([&inv](CNode* node) { node->PushInventory(inv); });
+        const TransactionError err = BroadcastTransaction(tx, err_string, max_tx_fee, relay, /*wait_callback*/ false, /*check_chain*/ false);
```

Comment:
Sorry, I think I got this wrong in my patch

```suggestion
        const TransactionError err = BroadcastTransaction(tx, err_string, max_tx_fee, relay, /*check_chain*/ false, /*wait_callback*/ false);
```